packages/frontend/src/features/comments/hooks/useDashboardCommentsCheck.ts

```diff
@@ -15,9 +15,9 @@ export const useDashboardCommentsCheck = (
         'DashboardComments',
     );
 
-    const isDashboardCommentsEnabled = useFeatureFlagEnabled(
-        FeatureFlags.DashboardComments,
-    );
+    // We want to keep this flag enabled by default, so users on self-hosting can use this feature
+    const isDashboardCommentsEnabled =
+        useFeatureFlagEnabledPosthog(FeatureFlags.DashboardComments) !== false;
```

Comment:
I could also modify `useFeatureFlagEnabled` to return a `default` if the flag returns undefined .